Functions.Templates/Bindings/bindings.json

```diff
@@ -1747,6 +1747,630 @@
+                  {
+                    "value": "Updated",
+                    "display": "Updated"
+                  },
+                  {
+                    "value": "Deleted",
+                    "display": "Deleted"
+                  }
+                ],
+                "required": true,
+                "label": "$MSGraphWebhook_in_changeType_label",
+                "help": "$MSGraphWebhook_in_changeType_help"
+              },
+              {
+                "name": "Listen",
+                "value": "enum",
+                "enum": [
+                  {
+                    "value": "me/mailFolders('Inbox')/Messages",
+                    "display": "Email Messages"
+                  },
+                  {
+                    "value": "me/drive/root",
+                    "display": "OneDrive Root"
+                  },
+                  {
+                    "value": "me/events",
+                    "display": "Calendar events"
+                  },
+                  {
+                    "value": "me/contacts",
+                    "display": "Contacts"
+                  }
+                ],
+                "label": "$MSGraphWebhook_in_listen_label",
+                "help": "$MSGraphWebhook_in_listen_help"
+              },
+              {
+                "name": "PrincipalId",
+                "value": "token",
+                "resource": "MSGraph",
+                "required": false,
+                "label": "$principal_id_label",
+                "help": "$principal_id_help"
+              },
+              {
+                "name": "idToken",
+                "value": "string",
+                "required": false,
+                "label": "$idToken_label",
+                "help": "$idToken_help"
+              }
+            ],
+            "rules": [
+              {
+                "name": "disable",
+                "type": "NAND",
+                "values": [
+                  { "value": "idToken" },
+                  { "value": "PrincipalId" }
+                ]
+              }
+            ]
+      },
+      {
+            "type": "GraphWebhookTrigger",
+            "direction": "trigger",
+            "displayName": "$MSGraphWebhookTrigger_displayName",
+            "documentation": "$content=Documentation\\GraphWebhookTrigger.md",
+            "enabledInTryMode": false,
+            "settings": [
+                {
+                    "name": "name",
+                    "value": "string",
+                    "defaultValue": "notification",
+                    "required": true,
+                    "label": "$MSGraphWebhookTrigger_in_name_label",
+                    "help": "$MSGraphWebhookTrigger_in_help_label"
+                },
+                {
+                    "name": "Type",
+                    "value": "enum",
+                    "defaultValue": "#Microsoft.Graph.Message",
+                    "required": true,
+                    "isHidden": true,
```

Comment:
(As discussed before) this allows one input to set two properties in function.json. While this is "magic," it is documented, so I decided to leave it in for now.